source/renderer/app/stores/SidebarStore.js

```diff
@@ -84,7 +84,7 @@ export default class SidebarStore extends Store {
       [categories.STAKING_DELEGATION_COUNTDOWN.name]: false,
       [categories.STAKING.name]: true,
       [categories.SETTINGS.name]: true,
-      [categories.VOTING.name]: isMainnet || isDev,
+      [categories.VOTING.name]: isMainnet || isDev || isShelleyTestnet,
```

Comment:
@aleksandardjordjeviciohk I asked to enable voting for `isTestnet` and not `isShelleyTestnet`...
Please replace `isShelleyTestnet` with `isTestnet` and include `isTestnet` in this line to get its value: https://github.com/input-output-hk/daedalus/blob/da28eed137e22204ca2ccd5a677628ec860c2086/source/renderer/app/stores/SidebarStore.js#L71

Please let @gabriela-ponce know once you are done with this change so that she can continue with testing. Thanks!

Reply:
Nevermind. I did it here: https://github.com/input-output-hk/daedalus/pull/2470/commits/c02afb3587f278fefb5fb6d2ee0ec5699fb4fefa